src/app.js

```diff
@@ -37,18 +38,18 @@ async function getCountry(countries) {
 
```

Comment:
voorheen had ik in het rechterrijtje staan dat de countries[i] overeen moest komen met de sortedCountries[j]
Dus dan pusht hij steeds countries[i] in de nieuwe array terwijl dit niet de gesorteerde code is!
Die heb ik even snel aangepast en zoals die hieronder staat klopt ie helemaal!